lib/koans/05_maps.ex

```diff
@@ -14,8 +14,8 @@ defmodule Maps do
   end
 
   koan "A map has keys and values" do
```

Comment:
I would be okay with deleting this koan. It feels very clunky in the current form.